### PR TITLE
Update docs sidebar UI and other UI fixes

### PR DESCRIPTION
**Pull request by DonOmalVindula**
## Purpose
Implements new sidebar UI for table of contents, left-side navbar and fixes numerous UI issues in on-prem docs site.

## Goals
This PR replaces the existing sidebar UIs of the on-prem documentation site with a newer design and fixes issues mentioned below.

Resolves https://github.com/wso2/product-is/issues/11782
Resolves https://github.com/wso2/product-is/issues/11785
Resolves https://github.com/wso2/product-is/issues/11808
Resolves https://github.com/wso2/product-is/issues/11850
Resolves https://github.com/wso2/product-is/issues/11851

## Approach
1. The left navigation sidebar UI was implemented by updating the `nav.html` and `nav-item.html` in `en/theme/material/partials/` directory.

2. The right side table of contents was implemented by updating the `toc.html` and `toc-item.html` in `en/theme/material/partials/` directory.

3.  Footer and header was changed to accommodate the new sidebars by altering files `footer.html`, `tabs.html` and `tabs-item.html` in `en/theme/material/partials/` directory.

4. Added a new icon (arrow_down.svg) for the left sidebar UI in `en/docs/assets/img/icons/` directory.

5. `en/docs/assets/css/theme.css`, `en/theme/material/templates/landing-page.html` and `en/docs/assets/css/theme.css` files were altered to fix UI issues in Mac systems and mobile views.

image

## Test environment
Tested with the following Mkdocs dependencies on Ubuntu 20.04.1
- mkdocs - 1.1.2
- mkdocs-material - 4.1.1
- mkdocs-material-extensions - 1.0.1
- pymdown-extensions - 5.0

### CODE REVIEW

File: en/theme/material/partials/toc.html [hunk 1]
```diff
@@ -1,12 +1,29 @@
+<!--
+ * Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (kanchanagodage) [inline en/theme/material/partials/toc.html:2]: 2019 or 2021?

File: en/theme/material/templates/landing-page.html [hunk 2]
```diff
@@ -114,6 +104,23 @@
       .md-main .md-sidebar.md-sidebar--primary {
         width: 0px;
       }
+      .md-tabs__inner {
+        margin-right: 11vw !important;
+        margin-left: 11vw !important;
+        max-width: unset !important;
+        padding: 0px !important;
+      }
```
Comment (kanchanagodage) [inline en/theme/material/templates/landing-page.html:112]: Format the code with 4 space indentation

File: en/theme/material/templates/landing-page.html [hunk 3]
```diff
@@ -19,101 +19,139 @@
...
-      .md-main .md-sidebar.md-sidebar--primary {
-        width: 0px;
-      }
-    }    
-  </style>
+{{ super() }}
+<style>
+	/* Overriding default CSS set by mkdocs without affecting other components using the md-main__inner class */
+	.md-main .md-content {
+		margin-right: 0;
```
Comment (NipuniBhagya) [inline en/theme/material/templates/landing-page.html:26]: Please fix the indentation here.

File: en/theme/material/partials/toc.html [hunk 4]
```diff
@@ -1,12 +1,29 @@
+<!--
```
Comment (NipuniBhagya) [inline en/theme/material/partials/toc.html:1]: If this file is from the material theme we can't add our license header. Please verify it.

File: en/docs/assets/img/icons/landing-page/arrow_down.svg [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+-->
+
+<svg xmlns="http://www.w3.org/2000/svg" height="18px" viewBox="0 0 24 24" width="18px" fill="#000000"><path d="M24 24H0V0h24v24z" fill="none" opacity=".87"/><path d="M16.59 8.59L12 13.17 7.41 8.59 6 10l6 6 6-6-1.41-1.41z"/></svg>
```
Comment (NipuniBhagya) [inline en/docs/assets/img/icons/landing-page/arrow_down.svg:19]: Please format the SVG properly. Also please add an ID to it.

**Review by NipuniBhagya — CHANGES_REQUESTED** [review]

File: en/theme/material/templates/landing-page.html [hunk 6]
```diff
@@ -19,101 +19,143 @@
...
-    @media (min-width: 1220px) {
-      .md-main .md-sidebar.md-sidebar--primary {
-        width: 0px;
-      }
-    }    
-  </style>
-{% endblock %}
+  }
+</style>
+{% endblock %}
```
Comment (NipuniBhagya) [inline en/theme/material/templates/landing-page.html:161]: Add a new line here.

**Review by NipuniBhagya — APPROVED** [review]

**Pull request merged**